helperApp/plaid/plaid.scss

```diff
@@ -1 +1,10 @@
-@import "../assets/css/blockchain.css.scss";
+
+@import "../../bower_components/bootstrap-sass/assets/stylesheets/bootstrap/variables";
+@import "../../bower_components/bootstrap-sass/assets/stylesheets/bootstrap/mixins";
+@import "../../bower_components/bootstrap-sass/assets/stylesheets/bootstrap/buttons";
+@import "../../assets/css/utilities/colors";
+@import "../../assets/css/elements/buttons";
+
```

Comment:
I did that mostly because I was having trouble configuring webpack to handle fonts. Will have to cross that bridge eventually though.